src/scenes/AcceleratorRouter/AcceleratorCohortEditView.tsx

```diff
@@ -0,0 +1,106 @@
+import { Typography, useTheme } from '@mui/material';
+import React, { useCallback, useEffect, useState } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { APP_PATHS } from 'src/constants';
+import AcceleratorMain from 'src/scenes/AcceleratorRouter/AcceleratorMain';
+import CohortForm from 'src/scenes/AcceleratorRouter/CohortForm';
+import CohortService from 'src/services/CohortService';
+import strings from 'src/strings';
+import { UpdateCohortRequestPayload } from 'src/types/Cohort';
+import useForm from 'src/utils/useForm';
+import useSnackbar from 'src/utils/useSnackbar';
+
+type AcceleratorCohortEditViewProps = {
+  reloadData?: () => void;
+};
+
+export default function AcceleratorCohortEditView({ reloadData }: AcceleratorCohortEditViewProps): JSX.Element {
+  const history = useHistory();
+  const theme = useTheme();
+  const [isBusy, setIsBusy] = useState<boolean>(false);
```

Comment:
After you hook up the async thunks, this can be replaced by looking at `request.status === 'pending'`